ckanext/unaids/custom_user_profile/actions.py

```diff
@@ -0,0 +1,157 @@
+import logging
+
+from flask_babel import force_locale
+
+import ckan.authz as authz
+from ckan import model, logic
+from ckan.common import _
+from ckan.lib.helpers import url_for, flash_success
+from ckan.lib.mailer import mail_user
+from ckan.plugins import toolkit
+from ckan.plugins.toolkit import config
+from ckanext.ytp_request.helper import get_safe_locale
+import ckanext.ytp_request.logic.action.create as ytp_request_action_create
+from ckanext.ytp_request.model import MemberRequest
+
+log = logging.getLogger(__name__)
+
+_SUBJECT_MEMBERSHIP_REQUEST = _("New membership request (%(organization)s)")
+_MESSAGE_MEMBERSHIP_REQUEST = _("""\
+    User %(user)s (%(email)s), %(job_title)s at %(affiliation)s, has requested membership to organization %(organization)s.
+
+    %(link)s
+
+    Best wishes,
+    The AIDS Data Repository
+    """)
+
+
+def mail_new_membership_request(locale, admin, group_name, url, user_name, user_email, user_job_title, user_affiliation):
+    """
+    Overrides the mail_new_membership_request function from ckanext-ytp-request plugin
+    """
+    with force_locale('en'):
+        subject = _SUBJECT_MEMBERSHIP_REQUEST % {
+                      'organization': group_name
+                  }
+        message = _MESSAGE_MEMBERSHIP_REQUEST % {
+                      'user': user_name,
+                      'email': user_email,
+                      'organization': group_name,
+                      'link': url,
+                      'job_title': user_job_title,
+                      'affiliation': user_affiliation
+                  }
+    try:
+        mail_user(admin, subject, message)
+    except Exception:
+        log.exception("Mail could not be sent")
+
+
+def _create_member_request(context, data_dict):
+    """
+    Whole method copied over from ckanext-ytp-request plugin
+    Helper to create member request
```

Comment:
I thought we had decided since it was our fork and no longer actively maintained upstream we could just edit ytp_request code in place? This approach of overriding from ckanext-unaids is creating significant amounts of duplicate code?